### DISCUSSION TITLE
Account recovery request link expired

### DISCUSSION THREAD

**Discussion by reghuramk** [community/community#40948] [New to GitHub]
Hi I'm Reghuram Karunamurthi,

My phone got factory reset last month, since I enabled 2FA for the login, I lost my access to that app and now I'm unable to login to my account. I used my GitHub recovery codes but none of the codes worked. So I requested account recovery and the account recovery request got approved. It was my bad that I didn't notice that email and the link expired after 72 hours. I'm feeling extremely helpless right now as I use my account not only for my personal work but also for my professional work organization. I have again tried requesting my account recovery but cannot proceed further this time. Kindly help me recover my account as I face terrible consequences at my workplace if I fail to recover it. I have already emailed GitHub support regarding this. Is there an alternate solution to this problem?

PS: I accessed this GitHub community discussion from the account that I've already logged in to from my work laptop. I tried disabling the 2FA but it asks for my password and again it leads me back to the same 2FA to which I don't have access. This account would get locked out regularly and we need to access this account through 2FA for security reasons - my organization made this mandatory.

Thanks in advance, Cheers.

![ScreenShot](https://user-images.githubusercontent.com/54064162/205984999-422d9f21-74b8-492c-ab9a-6c34248a28f6.png)

**Comment by gianlucajahn** [comment]
I'm sorry to hear that you're having trouble accessing your GitHub account. If you've already tried using your GitHub recovery codes and they didn't work, and you've also requested account recovery but the link has expired, it's possible that your only option at this point is to continue working with GitHub support to try to recover your account.

When you request account recovery, GitHub sends an email to the email address associated with your account. This email includes a link that you can use to reset your password and regain access to your account. However, this link is only valid for 72 hours, after which it expires and can no longer be used. If you didn't receive the email or if the link has expired, you can try requesting account recovery again by following these steps:

Go to the GitHub login page and click the "I forgot my password" link.
Enter the email address associated with your account and click the "Send password reset email" button.
Check your email for a message from GitHub with the subject "Reset your GitHub password."
Click the link in the email to reset your password and regain access to your account.
If you're still unable to access your account after requesting account recovery, you can contact GitHub support for further assistance. They should be able to help you recover your account and reset your password.

As for your question about disabling 2FA, you can disable two-factor authentication (2FA) for your account by following these steps:

Go to your GitHub settings page.
On the left side of the page, click the "Security" tab.
Under the "Two-Factor Authentication" section, click the "Turn off two-factor authentication" button.
Enter your password and click the "Turn off two-factor authentication" button to confirm.
Keep in mind that disabling 2FA will make your account less secure, so it's generally not recommended unless you have no other option. In your case, it may be best to work with GitHub support to recover your account and reset your password, rather than disabling 2FA.

I hope this helps. Good luck with recovering your account and please feel free to ask if you have any further questions.

**Reply by reghuramk** [reply]
Thanks for replying @gianlucajahn, as you mentioned in your reply, I tried to reset my password, the reset link got sent to my email address. Clicking that link once again led me to enter one of my recovery codes or the 2FA. Thanks for trying to figure out the solution though. Appreciate it.

**Reply by queenofcorgis** [reply]
Hi @Reghuramk! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by reghuramk** [reply]
Hi, @queenofcorgis, I have already raised a ticket regarding this and I haven't gotten any response from the GitHub support yet. Hope to get the issue resolved. Thanks!
